packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/components/offers/translations/Messages_fr_FR.json

```
@@ -2,15 +2,7 @@
   "dedicatedCloud_offer_classic": "Veeam Backup Standard",
   "dedicatedCloud_offer_premium": "Veeam Backup Premium",
   "dedicatedCloud_offer_advanced": "Veeam Backup Advanced",
-  "dedicatedCloud_offer_upgrade_custom_report": "Rapport de sauvegarde personnalisable",
-  "dedicatedCloud_offer_upgrade_encrypted_backup": "Sauvegarde chiffrée",
-  "dedicatedCloud_offer_upgrade_retention_points": "{{retention}} points de rétention",
-  "dedicatedCloud_offer_upgrade_full_backups": "{{fullBackups}} sauvegardes complètes",
-  "dedicatedCloud_offer_upgrade_replication": "Réplication hors site",
-  "dedicatedCloud_offer_upgrade_choice_backup_days": "Choix des jours pour chaque sauvegarde",
-  "dedicatedCloud_offer_upgrade_choice_backup_time": "Choix de l’heure de début des sauvegardes",
-  "dedicatedCloud_offer_upgrade_long_term_retention": "11 points de rétention longue durée (GPS rétention)",
-  "dedicatedCloud_offer_upgrade_proxy_backup": "1 backup proxy par hôte",
   "dedicatedCloud_offer_current_offer": "Offre actuelle",
-  "dedicatedCloud_offer_price_per_vm": "/VM"
+  "dedicatedCloud_offer_price_per_vm": "/VM",
```

```suggestion
  "dedicatedCloud_offer_price_per_vm": "{{price}}/VM",
```

or else the translators won't be able to understand what is going on here

---

Price is a component. This is the text after displaying price.